hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/src/test/java/org/apache/hadoop/mapred/TestJobClient.java

```diff
@@ -22,77 +22,81 @@
 import java.util.Collection;
 
 import org.apache.hadoop.conf.Configuration;
+
+import static org.junit.jupiter.api.Assertions.*;
+
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.FileUtil;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.mapred.ClusterStatus.BlackListInfo;
 import org.apache.hadoop.mapreduce.MRConfig;
 import org.apache.hadoop.mapreduce.server.jobtracker.JTConfig;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
 
 public class TestJobClient {
 
   final static String TEST_DIR = new File("target",
     TestJobClient.class.getSimpleName()).getAbsolutePath();
 
-  @After
+  @AfterEach
   public void tearDown() {
     FileUtil.fullyDelete(new File(TEST_DIR));
   }
 
   @Test
-  public void testGetClusterStatusWithLocalJobRunner() throws Exception {
+  void testGetClusterStatusWithLocalJobRunner() throws Exception {
     Configuration conf = new Configuration();
     conf.set(JTConfig.JT_IPC_ADDRESS, MRConfig.LOCAL_FRAMEWORK_NAME);
     conf.set(MRConfig.FRAMEWORK_NAME, MRConfig.LOCAL_FRAMEWORK_NAME);
     JobClient client = new JobClient(conf);
     ClusterStatus clusterStatus = client.getClusterStatus(true);
     Collection<String> activeTrackerNames = clusterStatus
         .getActiveTrackerNames();
-    Assert.assertEquals(0, activeTrackerNames.size());
+    assertEquals(0, activeTrackerNames.size());
     int blacklistedTrackers = clusterStatus.getBlacklistedTrackers();
-    Assert.assertEquals(0, blacklistedTrackers);
+    assertEquals(0, blacklistedTrackers);
     Collection<BlackListInfo> blackListedTrackersInfo = clusterStatus
         .getBlackListedTrackersInfo();
-    Assert.assertEquals(0, blackListedTrackersInfo.size());
+    assertEquals(0, blackListedTrackersInfo.size());
   }
 
-  @Test(timeout = 10000)
-  public void testIsJobDirValid() throws IOException {
```

Comment:
is removing public mandatory? if things can work with this and others staying public, Would prefer it stay as public only